### DISCUSSION TITLE
请教一下fyne国际化实现方式和重载实现方法

### DISCUSSION THREAD

**Discussion by ZhangSir-github** [he3-app/he3-feedback#20] [Q&A] [answered]
大佬是用go fyne框架写的吧，fyne目前框架还没支持国际化，最近也尝试了一下这个框架，国际化没想到好的实现方式，fyne默认使用系统默认字体，不同字体的显示又有很大差异。大佬可以大概讲下您是怎么做的国家化，怎么重载的呢。

**Comment by zhenweiwang1990** [comment, marked as answer]
我们是使用 Electron
